examples/README.md

```diff
@@ -0,0 +1,61 @@
+# Examples for Testing Features Locally
+
+The example projects are meant to be used to test features locally by contributors working on this SDK. In the `requirements.txt` file, the deepgram package used is the local version:
+
+```
+-e ../../
+```
+
+## Steps to Test Your Code
+
+If you are contributing changes to this SDK, you can test those changes by using the `prerecorded` or `streaming` projects in the `examples` folder. Here are the steps to follow:
+
+### Add Your Code
+Make your changes to the SDK (be sure you are on a branch you have created to do this work).
+
+### Install dependencies
+
+Run the following command to install the project dependencies:
+
+`pip install -r requirements.txt`
+
+### Edit the API key
+
+Replace the API key where it says 'YOUR_DEEPGRAM_API_KEY'
+
+`DEEPGRAM_API_KEY = 'YOUR_DEEPGRAM_API_KEY'`
+
+### Run the project
+
+Make sure you're in the directory with the `main.py` file and run the project with the following command.
+
+`python3 main.py`
```

Comment:
I really hope no one is using python 2 anymore 👀  I think we can change this to `python main.py`